### PR TITLE
Documentation Updates & Exploration Scripts

### PR DESCRIPTION
**Pull request by frankchen07**
1. Updated data documentation with comments on `dashboard_useraction` and `dashboard_activity`. Fixed some nit grammar and missing punctuation.

2. Regression exploration scripts:

    `bounty_regression_features.sql` - feature extraction on Metabase.
    `bounty_regression.py` - linear regression exploration.
    `bounty_regression.ipynb` - success rates regression exploration with visualizations.

3. SendGrid exploration: 

    `sendgrid_email_performance.py` - use SendGrid API, data munging to read and CTR rates.

### CODE REVIEW

File: marketing/adhoc-email-performance/sendGridEmailPerformance.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+# initialize client
+with open(os.path.join(os.path.realpath('/Users/fronk/.creds'), 'sendgrid_creds.json'), 'r') as f:
+    config = json.load(f)
+api_key = config['key']
+host = 'https://api.sendgrid.com'
+request_headers = {'Authorization': 'Bearer {0}'.format(api_key)}
+version = 3
+client = python_http_client.Client(
+	host=host,
```
Comment (mbeacom) [inline marketing/adhoc-email-performance/sendGridEmailPerformance.py]: tabs?! 😛

**Review by mbeacom — COMMENTED** [review]
@frankchen07 Would you be opposed to us setting up an automated linter on this repo like `gitcoinco/web` ?

Also, would you mind updating the file names to `snake_case` vs mixedCase?

File: marketing/adhoc-email-performance/sendGridEmailPerformance.py (hunk 1 above)
Comment (frankchen07) [reply marketing/adhoc-email-performance/sendGridEmailPerformance.py]: 😬 :slowly backs away into the bushes: - fixed :)

**Comment by frankchen07** [comment]
@mbeacom - I would not be opposed to setting up automated linter on this repo. In my experience, data repos can get notoriously messy (adhoc scripts here and there, random research), and I intend for Gitcoin's to not be like that. 👍 

Not sure if you guys are using any kind of set naming schema, but I tend to explicitly label adhocs and also which pieces of code correspond to which repo and which issue.

**Comment by mbeacom** [comment]
@frankchen07 🙌 Base `setup.cfg` and stickler-ci configurations added in https://github.com/gitcoinco/data-ops/pull/22 - Once you update this branch with `master` the linting config changes should be reflected on the PR.

File: .stickler.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+linters:
```
Comment (mbeacom) [inline .stickler.yml]: @frankchen07 Looks like you might have a bad rebase here.  These already exist in `master` 🤔

**Pull request merged**